lms/djangoapps/courseware/tests/test_grades.py

```diff
@@ -68,7 +68,7 @@ def test_nonexistent_course(self):
 
     def test_all_empty_grades(self):
         """No students have grade entries"""
-        all_gradesets, all_errors = self._gradesets_and_errors_for(self.course.id, self.students)
+        all_gradesets, all_errors = self._gradesets_and_errors_for(self.course.id, self.students, keep_raw_scores=True)
```

Comment:
Why did you change `keep_raw_scores` to True? It doesn't look to me like `_gradesets_and_errors_for` uses the raw scores return value from iterate_grades_for.